src/Compilers/CSharp/Test/Symbol/Symbols/Source/RecordTests.cs

```diff
@@ -1130,7 +1130,7 @@ public static void Main()
                 Diagnostic(ErrorCode.ERR_MultipleRecordParameterLists, "(int X)").WithLocation(13, 17)
                 );
 
-            Assert.Equal(new[] { "C..ctor(System.Int32 X, System.Int32 Y)", "C..ctor(C )" }, comp.GetTypeByMetadataName("C")!.Constructors.Select(m => m.ToTestDisplayString()));
+            Assert.Equal(new[] { "C..ctor(System.Int32 X, System.Int32 Y)", "C..ctor(C original)" }, comp.GetTypeByMetadataName("C")!.Constructors.Select(m => m.ToTestDisplayString()));
```

Comment:
Would it make sense to include a test which asserts that a simple positional record has *no* members containing parameters with an empty string name?

Reply:
I think this scenario may be adequately addressed by existing tests which verify the symbol display of all members in the record. I am just thinking of a situation where additional synthesized members are introduced later, it is easiest to just name any parameters at the same time the new member is introduced.

Reply:
I agree that we should name parameters if we add any.
I'm pretty sure scenario with no positional members is covered already. Thanks